packages/gateway/src/features/registration/schema.graphql

```diff
@@ -422,14 +423,14 @@ input CertificateInput {
   collector: RelatedPersonInput
   hasShowedVerifiedDocument: Boolean
   payments: [PaymentInput]
-  data: String
+  certTemplateId: String
 }
 
 type Certificate { # -> Document Reference
   collector: RelatedPerson # -> .extension
   hasShowedVerifiedDocument: Boolean # -> .extension
   payments: [Payment] # -> .extension
-  data: String # -> .content.attachment.data base64
+  certTemplateId: String
```

Comment:
Mark these as required `certTemplateId: String!`. GraphQL should really treat these required by default unless marked otherwise

Reply:
What happens to old Certificates btw as the don't have this defined?

Reply:
I was thinking of taking care of this issue in query/registration-mapping. If any certificate doesn't have this, they will be assigned to a default template id.

Reply:
The mappings will unfortunately be removed when we build events v2. Is there a specific reason for not writing a migration for this? 
We should aim to always keep data as consistent as possible on the database level as the data quality really affects all parts of the stack.

Reply:
okay, will handle this in migration.

Reply:
One more thing regarding the migration. We get certificate config data from country-config. Now, in migrations, how and from where should we fetch this data to update existing documentReferences?
I don't see any pattern in existing migrations to fetch data from country-config or from other places.